Frontend/CO.CDP.OrganisationApp.Tests/Pages/Supplier/ConnectedEntity/ConnectedEntityCompanyRegistrationDateTest.cs

```diff
@@ -7,17 +7,17 @@
 
 namespace CO.CDP.OrganisationApp.Tests.Pages.Supplier.ConnectedEntity;
 
-public class ConnectedEntityCompanyRegistrationDateTest
+public class ConnectedEntityCompanyInsolvencyDateTest
```

Comment:
@shilpigoel-goaco Just wanted to double check that you intended to replace all Registration Date instances with Insolvency date? I would have expected these tests would be in a ConnectedEntityCompanyInsolvencyDateTest.cs file ?

Reply:
corrected